modules/ztp-creating-the-policygentemplate-cr.adoc

```diff
@@ -26,6 +26,23 @@ Depending on the specific requirements of your clusters, you might need more tha
 ====
 
 . Define all the policy namespaces in a YAML file similar to the example `out/argocd/example/policygentemplates/ns.yaml` file.
++
+[IMPORTANT]
+====
+Ensure that the namespaces meets the following requirements:
+
+. Namespaces must be prefixed with `ztp`. For example:
```

Comment:
Confirm: `ztp` or `ztp-`

Reply:
The constraint is that it must start with `ztp`. By convention we almost always use "ztp-...." but "ztpmypolicies" would be valid too.